src/lib/CoreInstance.cc

```diff
@@ -149,6 +149,9 @@ void CoreInstance::createL1InstructionMemory(
 
 void CoreInstance::setL1InstructionMemory(
```

Comment:
Is there a comment anywhere explaining why coreInstance is needed e.g. because of SST. If not it would be useful to explain this design decision e.g. at the top of coreInstance.hh

Reply:
I think @Jack said documentation will be added later, if i remember correctly

Reply:
and updates on this?

Reply:
Original comment out of scope.